1209/ABCD/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: CodeBridge
+owner: Intelectron India
+license: OSHW
```

Comment:
OSHW isn't a license; please specify a specific license that your firmware and hardware designs are under, and include the LICENSE file for that license in your code repository.